EIPS/eip-introspection.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: TBD
+title: Introspection precompiles
+description: Introspection precompiles that expose the current block context to the EVM
+author: Brecht Devos (@Brechtpd)
+discussions-to: TBD
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5